ui/src/modules/leads/containers/filters/StatusFilter.tsx

```diff
@@ -0,0 +1,18 @@
+import StatusFilter from 'modules/leads/components/StatusFilter';
+import React from 'react';
+import { withProps } from '../../../common/utils';
+import { Counts } from '../../types';
+
+type Props = {
+  counts: Counts;
+};
+
+type FinalProps = {} & Props;
+
+class StatusFilterContainer extends React.Component<FinalProps> {
```

Comment:
convert it to a PureComponent or functional component as it does nothing but rendering